src/zenml/integrations/mlflow/model_registries/mlflow_model_registry.py

```diff
@@ -0,0 +1,779 @@
+                    version=version,
+                    description=description,
+                )
+            except MlflowException as e:
+                raise RuntimeError(
+                    f"Failed to update the description of model version "
+                    f"'{name}:{version}' in the MLflow model registry: {str(e)}"
+                )
+        # Update the model tags.
+        if tags:
+            try:
+                for key, value in tags.items():
+                    self.mlflow_client.set_model_version_tag(
+                        name=name,
+                        version=version,
+                        key=key,
+                        value=value,
+                    )
+            except MlflowException as e:
+                raise RuntimeError(
+                    f"Failed to update the tags of model version "
+                    f"'{name}:{version}' in the MLflow model registry: {str(e)}"
+                )
+        # Remove the model tags.
+        if remove_tags:
+            try:
+                for key in remove_tags:
+                    self.mlflow_client.delete_model_version_tag(
+                        name=name,
+                        version=version,
+                        key=key,
+                    )
+            except MlflowException as e:
+                raise RuntimeError(
+                    f"Failed to remove the tags of model version "
+                    f"'{name}:{version}' in the MLflow model registry: {str(e)}"
+                )
+        # Update the model stage.
+        if stage:
+            try:
+                self.mlflow_client.transition_model_stage(
+                    name=name,
+                    version=version,
+                    stage=stage.value,
+                )
+            except MlflowException as e:
+                raise RuntimeError(
+                    f"Failed to update the current stage of model version "
+                    f"'{name}:{version}' in the MLflow model registry: {str(e)}"
+                )
+        return self.get_model_version(name, version)
+
+    def get_model_version(
+        self,
+        name: str,
+        version: str,
+    ) -> ModelVersion:
+        """Get a model version from the MLflow model registry.
+
+        Args:
+            name: The name of the model.
+            version: The version of the model.
+
+        Raises:
+            RuntimeError: If mlflow fails to get the model version.
+
+        Returns:
+            The model version.
+        """
+        # Get the model version from the MLflow model registry.
+        try:
+            mlflow_model_version = self.mlflow_client.get_model_version(
+                name=name,
+                version=version,
+            )
+        except MlflowException as e:
+            raise RuntimeError(
+                f"Failed to get model version '{name}:{version}' from the "
+                f"MLflow model registry: {str(e)}"
+            )
```

Comment:
Same as for `get_model`, there should be a way to look at the exception message to tell if it's a "model not found" or "version not found" error or something else, but I don't how what that is here:

```suggestion
            raise KeyError(
                f"Failed to get model version '{name}:{version}' from the "
                f"MLflow model registry: {str(e)}"
            )
```